### PR TITLE
data: Chrisidoko

### PR DESCRIPTION
**Pull request by Chrisidoko**

### CODE REVIEW

**Review by rahulpandey70 — DISMISSED** [review]
LGTM

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

File: data/Chrisidoko.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+   "name":"Eddie Jaoude",
```
Comment (eddiejaoude) [inline data/Chrisidoko.json:2]: I don't think this is correct?

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Incorrect name

File: data/Chrisidoko.json (hunk 1 above)
Comment (Chrisidoko) [reply data/Chrisidoko.json:2]: Yes i am aware i will make corrections to it

Comment (Chrisidoko) [reply data/Chrisidoko.json]: Done

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good, Thanks! 🎉

**Pull request merged**